### DISCUSSION TITLE
Getting following error when trying to connect from janusgraph to kerberised HBase server

### DISCUSSION THREAD

**Discussion by nikita15p** [JanusGraph/janusgraph#2970] [Q&A]
kinit by command line with same krb5.conf works but from janusgraph it failed on local exception: javax.security.sasl.SaslException: Failure to initialize security context [Caused by GSSException: Invalid name provided (Mechanism level: KrbException: Cannot locate default realm)]

Tested with janusgraph 0.6.0

**Comment by vtslab** [comment]
Try -Djava.security.krb5.conf=/etc/krb5.conf when starting your java application (or add it to the JAVA_OPTIONS when using gremlin client or gremlin server). Possibly, java kerberos will also find the krb5.conf when it is on the CLASSPATH.
